src/Exception/ExceptionConverter.php

```diff
@@ -51,6 +52,13 @@ public static function handleDownloadException(\Throwable $e): void
             ));
         }
 
+        // Catch user_error from phpseclib
+        // phpcs:disable
+        if (preg_match_all('/(getaddrinfo failed)|(Could not connect to)|(Cannot connect to)|(Root is invalid)|(The authenticity of)|(Connection closed prematurely)/', $e->getMessage())) {
```

Comment:
tady potom bude zbytečný `Could not connect to `, `Root is invalid`

Reply:
ty message jsou odchytnutý právě `FlysystemException` v PR
https://github.com/thephpleague/flysystem/pull/1070/files

Reply:
mas pravdu, tie som mal tiez odstranit. idem to fixnut.

Reply:
zmene v: https://github.com/keboola/ex-ftp/pull/36/commits/5f5ad3b12d127cc2d54692ded99605df59b976c7